client/flutter/lib/module_account/ui/screen/account_screen.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:injectable/injectable.dart';
+import 'package:twaslna_delivery/module_account/ui/state/account/account_state.dart';
+import 'package:twaslna_delivery/module_account/ui/state/account/account_loaded_state.dart';
+
+@injectable
+class AccountScreen extends StatefulWidget {
+
+  AccountScreen();
+
+  @override
+  AccountScreenState createState() => AccountScreenState();
+}
+
+class AccountScreenState extends State<AccountScreen> {
+  late AccountState currentState;
+
+  void refresh() {
```

Comment:
what use case of this?

Reply:
We are using refresh method to access setState 
late refer to this variable will have a value later